spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -100,15 +102,47 @@ def get_message(self):
 
         return message
 
+    @staticmethod
+    def get_tags(message):
+        tags = {}
+        failed_monitors = message.get("failed_monitors", [])
+        failed_monitors = [y.split("/") for y in failed_monitors]
+        for key, group in groupby(failed_monitors, key=lambda x: x[0]):
+            for mon in group:
+                try:
+                    k = mon[1].lower().replace(" ", "_")
+                    # 32 chars undocumented tag key length
+                    tags.update({k[:32]: 1})
```

Comment:
**question**: What if I have two methods with the same name (at least in the first 32 characters)?

Reply:
Yeah, there will be only one tag then. Note that this uses human-readable names if available so there is probably more variety. I also wonder if should strip leading "test_" to save some space.

Reply:
**suggestion**: If you decorate your monitor class and/or your monitor methods  with `@monitors.name`, the `test_` part will not be added, so it won't avoid duplication. I think it is worthwhile to add a warning about this limitation in [Sentry action](https://spidermon.readthedocs.io/en/latest/actions.html#sentry-action) documentation and the user can take care of not using long names for its monitor.

**suggestion**: Given that the format of the `failed_monitors` will always be <ClassName>/<MethodName>` (even if the names are defined by the user) I think `get_tags` could be simplified to:

```
    def get_tags(self, message):
        tags = {
            "spider_name": message.get("spider_name", ""),
            "project_name": self.project_name,
        }

        for failed_monitor in message.get("failed_monitors", []):
            # 32 chars limited by Sentry
            key = slugify(failed_monitor.split("/")[-1])[:32]
            tags[key] = 1

        return tags
```